capstone-frontend/src/components/CreateList.jsx

```diff
@@ -35,37 +35,60 @@ class CreateList extends Component {
       method: "GET",
     }).then(resp => resp.json());
 
-    document.location.href = `/listpage/${createdBookList[0].id}`;
+    this.setState({ loading: false, showModal: false, renderModal: false })
+
+    this.props.history.push(`/listpage/${createdBookList[0].id}`);
 
-    this.setState({ loading: false })
+    this.props.updateBookLists();
   }
 
   render() {
     return (
-      <div>
-        <h1 className="text-center mt-4"> Create Booklist </h1>
-
-        <Form onSubmit={this.handleSubmit}>
-          <Form.Group controlId="createBookList">
-            <Form.Label>Name of Booklist</Form.Label>
-            <Form.Control type="text" placeholder="Enter Name" />
-          </Form.Group>
-          <Button variant="primary" type="submit" disabled={this.state.loading}>
-            Create Booklist
-            {this.state.loading &&
-              <Spinner
-                as="span"
-                animation="border"
-                size="sm"
-                role="status"
-                aria-hidden="true"
-                className="ml-4"
-              />}
-          </Button>
-        </Form>
-      </div>
+      <>
```

Comment:
Is there any benefit to this wrapper element just being an empty element instead of a div?

Reply:
I meant to comment on the `<> </>` vs `<div> </div>`

Reply:
Not really, I will change it to a div if that is the standard